package-lock.json

```diff
@@ -883,7 +883,7 @@
         "source-map": {
           "version": "0.6.1",
           "resolved": "https://registry.npmjs.org/source-map/-/source-map-0.6.1.tgz",
-          "integrity": "sha512-UjgapumWlbMhkBgzT7Ykc5YXUT46F0iKu8SGXq0bcwP5dz/h0Plj6enJqjz1Zbq2l5WaqYnrVbwWOWMyF3F47g==",
+          "integrity": "sha1-dHIq8y6WFOnCh6jQu95IteLxomM=",
```

Comment:
Next time, please commit the result of the audit --fix in a separate commit. This way you can describe which package needed an upgrade and why. This information is now lost.

Reply:
I found the change to little to document, mainly because this was in Lodash and this isn't a dependency we use ourselves directly.

Reply:
I still think keeping commits atomic is worthwhile. For example your commit: 

1. Addresses the security issue
2. Installs yarn in able to run the tests
3. Enables the npm audit task

Three very much related tasks, but individual tasks nonetheless.